src/components/header/header-cell.component.ts

```diff
@@ -84,6 +84,10 @@ export class DataTableHeaderCellComponent {
 
   @HostBinding('attr.title')
   get name(): string {
+    // TODO review this.column.name logic. just return this.column.name?
+    // column name is guaranteed by setColumnDefaults()
+    // so this.column.prop is probably never returned
+    // However, if name is not truthy, this.column.prop could be a number and return
     return this.column.name || this.column.prop;
```

Comment:
Any reason not to change this to:
`return this.column.name`

Otherwise, to be safe (and conform to the API) it should be:
`return this.column.name || String(this.column.prop)`